### PR TITLE
WFLY-14061 Replace Java EE references in java doc with corresponding Jakarta EE ones (overall check)

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-14061
Link with: https://github.com/wildfly/wildfly/pull/13808

### CODE REVIEW

File: connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties [hunk 1]
```diff
@@ -80,7 +80,7 @@ datasources.data-source.idle-timeout-minutes=The idle-timeout-minutes elements s
 datasources.data-source.interleaving=An element to enable interleaving for XA connections
 datasources.data-source.jndi-name=Specifies the JNDI name for the datasource
 
-datasources.data-source.jta=Enable JTA integration
+datasources.data-source.jta=Enable Jakarta Persistence integration
```
Comment (bstansberry) [inline connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties:83]: This is Jakarta Transactions, not Jakarta Persistence.

File: connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties [hunk 2]
```diff
@@ -95,7 +95,7 @@ datasources.data-source.credential-reference.type=The type of credential this re
 datasources.data-source.credential-reference.alias=The alias which denotes stored secret or credential in the store
 datasources.data-source.credential-reference.clear-text=Secret specified using clear text (check credential store way of supplying credential/secrets to services)
 datasources.data-source.new-connection-sql=Specifies an SQL statement to execute whenever a connection is added to the connection pool
-datasources.data-source.no-tx-separate-pool=Oracle does not like XA connections getting used both inside and outside a JTA transaction. To workaround the problem you can create separate sub-pools for the different contexts
+datasources.data-source.no-tx-separate-pool=Oracle does not like XA connections getting used both inside and outside a Jakarta Persistence transaction. To workaround the problem you can create separate sub-pools for the different contexts
```
Comment (bstansberry) [inline connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties:98]: This is Jakarta Transactions, not Jakarta Persistence.

File: connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties [hunk 3]
```diff
@@ -216,7 +216,7 @@ datasources.xa-data-source.idle-timeout-minutes=The idle-timeout-minutes element
 datasources.xa-data-source.interleaving=An element to enable interleaving for XA connections
 datasources.xa-data-source.jndi-name=Specifies the JNDI name for the datasource
 
-datasources.xa-data-source.jta=Enable JTA integration
+datasources.xa-data-source.jta=Enable Jakarta Persistence integration
```
Comment (bstansberry) [inline connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties:219]: This is Jakarta Transactions, not Jakarta Persistence.

File: connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties [hunk 4]
```diff
@@ -232,7 +232,7 @@ datasources.xa-data-source.credential-reference.alias=The alias which denotes st
 datasources.xa-data-source.credential-reference.clear-text=Secret specified using clear text (check credential store way of supplying credential/secrets to services)
 
 datasources.xa-data-source.new-connection-sql=Specifies an SQL statement to execute whenever a connection is added to the connection pool
-datasources.xa-data-source.no-tx-separate-pool=Oracle does not like XA connections getting used both inside and outside a JTA transaction. To workaround the problem you can create separate sub-pools for the different contexts
+datasources.xa-data-source.no-tx-separate-pool=Oracle does not like XA connections getting used both inside and outside a Jakarta Persistence transaction. To workaround the problem you can create separate sub-pools for the different contexts
```
Comment (bstansberry) [inline connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties:235]: This is Jakarta Transactions, not Jakarta Persistence.

**Comment by RanabirChakraborty** [comment]
Sorry for this PR! Closing it.

File: connector/src/main/resources/org/jboss/as/connector/subsystems/resourceadapters/LocalDescriptions.properties [hunk 5]
```diff
@@ -90,7 +90,7 @@ connection-definitions.capacity-incrementer-properties=Properties to inject in c
 connection-definitions.capacity-decrementer-class=Class defining the policy for decrementing connections in the pool.
 connection-definitions.capacity-decrementer-properties=Properties to inject in class defining the policy for decrementing connections in the pool.
 connection-definitions.no-recovery=Specifies if the connection pool should be excluded from recovery.
-connection-definitions.no-tx-separate-pool=Oracle does not like XA connections getting used both inside and outside a JTA transaction. To workaround the problem you can create separate sub-pools for the different contexts.
+connection-definitions.no-tx-separate-pool=Oracle does not like XA connections getting used both inside and outside a Jakarta Persistence transaction. To workaround the problem you can create separate sub-pools for the different contexts.
```
Comment (bstansberry) [inline connector/src/main/resources/org/jboss/as/connector/subsystems/resourceadapters/LocalDescriptions.properties:93]: This is Jakarta Transactions, not Jakarta Persistence.

File: datasources-agroal/src/main/resources/org/wildfly/extension/datasources/agroal/LocalDescriptions.properties [hunk 6]
```diff
@@ -43,7 +43,7 @@ datasources-agroal.datasource.remove=Add operation for a datasource resource in
 #
 datasources-agroal.datasource.name=Name for the datasource (used for management)
 datasources-agroal.datasource.jndi-name=Specifies the JNDI name for the datasource
-datasources-agroal.datasource.jta=Enable JTA integration
+datasources-agroal.datasource.jta=Enable Jakarta Persistence integration
```
Comment (bstansberry) [inline datasources-agroal/src/main/resources/org/wildfly/extension/datasources/agroal/LocalDescriptions.properties:46]: This is Jakarta Transactions, not Jakarta Persistence.

**Pull request closed without merging**